app/assets/javascripts/snowcrash/pages/nodes/notes/table.js.coffee

```diff
@@ -0,0 +1,58 @@
+document.addEventListener "turbolinks:load", ->
+  if $('body.nodes.show').length
+
+    indexTable = new IndexTable('note')
+    
+    # # Checkbox behavior: select all, show 'btn-group', etc.
```

Comment:
what's the purpose of this commented code?

Reply:
removed